broker/src/main/java/io/camunda/zeebe/broker/transport/backupapi/BackupApiRequestHandler.java

```diff
@@ -170,6 +171,33 @@ private ActorFuture<Either<ErrorResponseWriter, BackupApiResponseWriter>> handle
     return result;
   }
 
+  private ActorFuture<Either<ErrorResponseWriter, BackupApiResponseWriter>>
+      handleDeleteBackupRequest(
+          final BackupApiRequestReader requestReader,
+          final BackupApiResponseWriter responseWriter,
+          final ErrorResponseWriter errorWriter) {
+    final ActorFuture<Either<ErrorResponseWriter, BackupApiResponseWriter>> result =
+        new CompletableActorFuture<>();
+    final var backupId = requestReader.backupId();
+    backupManager
+        .deleteBackup(backupId)
+        .onComplete(
+            (ignore, error) -> {
+              if (error == null) {
+                final BackupStatusResponse response =
+                    new BackupStatusResponse()
+                        .setBackupId(backupId)
+                        .setStatus(BackupStatusCode.DOES_NOT_EXIST)
```

Comment:
In both cases the end state of the backups is does_not_exist. If you strongly object to it, I can add a new response type 
 `BackupDeleteResponse` which does not contains a status.

Reply:
Yeah, from that perspective it makes sense :+1: 
If we compare it to other APIs, this behavior is often handled differently though: https://docs.github.com/en/rest/deployments/deployments?apiVersion=2022-11-28#delete-a-deployment--status-codes

No strong objection from me, I think there are arguments for both sides.

Reply:
Operate and optimize currently do not distinguish these two cases. They always returns same response even if the backup does not exist already.

Reply:
I will leave it as it is. The implementation also does not handle that case. If we want to handle the case when the backup does not exist, it will be returned as an error anyway.